.changeset/little-guests-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": minor
+---
```

Comment:
![image](https://github.com/user-attachments/assets/6fa01939-31c4-4c97-8266-9e7c3f666d38)
What's our stance on this particular situation? System style is set to dark, app µode is set to light.

Reply:
![image](https://github.com/user-attachments/assets/0e624076-c199-46d4-8763-69da74ac9723)
At first it's not clear that the two Select would display a different set of values, especially since the code is hidden by default, maybe setting a label would help the user open and understand which is which.

Reply:
![image](https://github.com/user-attachments/assets/e7c7406b-7581-4c27-9390-2816fd97be5b)
When pressed the radio border is not visible

Reply:
Are we expecting a double focus state? 
![image](https://github.com/user-attachments/assets/0c823753-2310-409b-9264-6b95989f20bb)

Reply:
lol the double focus may be a bit weird.. I fonder how to go about fixing this one.

Reply:
for the scrollbar color, we will create another ticket to fix this issue.

Reply:
> lol the double focus may be a bit weird.. I fonder how to go about fixing this one.

maybe with a tabIndex of -1 ?